target/snitch_cluster/README.md

```diff
@@ -244,8 +244,8 @@ The traces contain a lot of information which we might not be interested at firs
 
 ```
             , compute
-"range(0,9)",       1
-9           ,
+"range(0,8)",       1
```

Comment:
I am only just learning about the feature and format. But given the example, would it not make more sense to make it to illustrate the feature?

```
            , compute, idle   
"range(0,7)",       1,
8           ,        ,       1
```

Reply:
I think this would indeed make sense. Thanks for the suggestion @GiannaP :)

Reply:
After looking at the code however, the idle region would actually be very short (it doesn't include the barrier synchronization with the other cores).

To not complicate the code at this point I would postpone this demonstration. I anyways wanted to extend the tutorial to demonstrate how to use the DMA, and that would be perfect to also demonstrate the feature you suggested here.

Reply:
Yes, exactly. It looks like that because there is no barrier with the other cores:
![image](https://github.com/pulp-platform/snitch_cluster/assets/5629192/de1dedc5-cd97-4767-a537-d8db3e63033c)

Nevertheless, it gives a first functionality indication, and the DMA core is not computing anything in that phase, so it is still better fitting than calling it also a computation phase for the DMA core IMO. However, feel free to postpone this for future documentation.

Reply:
Indeed I agree, but in fact the currently modified `layout.csv` should not display anything for the DM core, i.e. neither "compute" nor "idle".

Reply:
Ah, in your new version, yes, indeed, it does not. I also just noticed Thread 7 is missing in my example. As mentioned, feel free to leave it for now.